src/components/Qr/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FC, ReactNode, useEffect, useState } from 'react';
+import clsx from 'clsx';
+import QrCode from 'qrcode';
+
+export interface QrProps {
+  /** Optional. Extra classNames you can pass. */
+  className?: string;
+  /** Optional. margin for QR component. */
+  margin?: number;
+  /** Required. The content for QR code. */
+  text: string;
+  /** Optional. width size. */
+  width?: number;
+}
+
+/**
+ * Qr component.
+ */
+const Qr: FC<QrProps> = ({ className, margin = 0, text, width = 140 }) => {
+  const [qr, setQr] = useState<ReactNode | null>(null);
+
+  useEffect(() => {
+    const generateQR = async (): Promise<void> => {
+      const url = await QrCode.toDataURL(text, {
+        color: {
+          dark: '#000000',
+          light: '#0000',
```

Comment:
hmm the light color is an invalid color code? should it be `light: #ffffff` instead?

Reply:
will need to change the tests if this was a mistake

Reply:
Hmm - seems it's OG Bucky, i'll change that in the next commit. - (good catch btw @jamessspanggg)
https://github.com/thenewboston-developers/Website/blob/development/src/components/Qr/index.tsx#L20